code/src/Create.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+
+export const Create = () => {
+    
+    const [text, addMessage] = useState("")
```

Comment:
Names of state and state handlers should have same base so it's easy to detect their pairs. In your case, correct naming would be :

```
const [text, setText] = useState("");
```

or 

```
const [message, setMessage] = useState("");
```

Reply:
Fixed